tt_metal/tools/profiler/op_profiler.hpp

```diff
@@ -251,6 +258,47 @@ inline json get_base_json(
     return j;
 }
 
+template <typename operation_t>
+inline json get_base_json(
+    uint32_t operation_id,
+    const typename operation_t::operation_attributes_t& operation_attributes,
+    const typename operation_t::tensor_args_t& tensor_args,
+    typename operation_t::tensor_return_value_t& tensor_return_value) {
+    ZoneScoped;
+    json j;
+    j["global_call_count"] = operation_id;
+
+    std::string opName = "device operation";
```

Comment:
This is hardcoded, operation_attributes has that actual name right?

Reply:
I will update this to use  the name of `operation_t`